packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -862,11 +856,11 @@ export const prepareVaultManagerKit = (
             userSeatPromise,
           ]);
 
-          trace(`VM LiqV after long wait`, proceeds);
+          trace(`LiqV after long wait`, proceeds);
```

Comment:
I notice that this function is async but the bulk of it is sync from here on. It would be easier to maintain and follow if here down was in its own sync function. e.g. `helper.liquidateFromAuction`.

not a blocker